sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   include(FetchContent)
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  set(UNIFIED_RUNTIME_TAG d6af758779db6eebdc419fd5e249302f566eb5de)
+  set(UNIFIED_RUNTIME_TAG 7d22f806c45872dd6326a123915fe3c75be25b0e)
```

Comment:
I think we should start having the discipline of updating only stable tags from unified runtime. So in this case, the next weekly tag containing this change. That way, we have always at least a standard practice of what we want to update and we know the changes we are bringing from unified-runtime are stable.

However, I understand the urgency of this fix.

@smaslov-intel : what do you think?

Reply:
Yes we should define validation+promotion process for accepting new UR versions. 

Currently I do not think there is any additional validation being done on UR weekly tags. So they are not necessarily more stable than other commits.

Reply:
I agree we need improved stability guarantees going forward, what can that UR standalone testing be?
Of course, updating SYCL to use newer UR will additionally run pre-commit SYCL testing.